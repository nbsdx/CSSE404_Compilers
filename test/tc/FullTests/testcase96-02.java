class Blub{
    public static void main(String[] args){}
}

class Blob{
    public Blob foo(int Blob){
	Blob blob = new Blob();
	return Blob;
    }
}