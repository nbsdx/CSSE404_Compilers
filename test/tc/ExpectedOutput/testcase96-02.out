Actual return type int of method foo does not match declared type Blob
